test/acceptance/framework/consul/consul_cluster.go

```diff
@@ -459,6 +463,52 @@ func createOrUpdateLicenseSecret(t *testing.T, client kubernetes.Interface, cfg
 	})
 }
 
+// configureSCCs creates RoleBindings that bind the default service account to cluster roles
+// allowing access tot he anyuid and privileged Security Context Constraints on OpenShift.
+func configureSCCs(t *testing.T, client kubernetes.Interface, cfg *config.TestConfig, namespace string) {
+	anyuidClusterRole := "system:openshift:scc:anyuid"
+	privilegedClusterRole := "system:openshift:scc:privileged"
+	anyuidRoleBinding := "anyuid-test"
+	privilegedRoleBinding := "privileged-test"
```

Comment:
Does it make sense for these to be `const`?

Reply:
I made the first two consts since they are the specific names that openshift creates. The latter two can be anything we want, so I'll keep them as vars.